whats_fresh/whats_fresh_api/forms.py

```diff
@@ -89,3 +90,50 @@ class Meta:
             'name': forms.TextInput(attrs={'required': 'true'}),
             'image': forms.FileInput(attrs={'required': 'true'})
         }
+
+
+class ThemeForm(forms.ModelForm):
+
+    class Meta:
+        model = Theme
+        exclude = []
+        widgets = {
+            'name': forms.TextInput(attrs={'required': 'true'}),
+            'background_color': forms.TextInput,
+            'foreground_color': forms.TextInput,
+            'logo': forms.FileInput,
+            'slogan': forms.TextInput,
+            'site_title': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'products': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'active': forms.TextInput
+
+        }
+
+
+class ThemeAdminForm(forms.ModelForm):
+
+    class Meta:
+        model = Theme
+        exclude = []
+
+    def clean(self):
+        cleaned_data = super(ThemeAdminForm, self).clean()
+        vendors_slug = cleaned_data.get('vendors_slug')
+        products_slug = cleaned_data.get('products_slug')
+        preparations_slug = cleaned_data.get('preparations_slug')
+        stories_slug = cleaned_data.get('stories_slug')
+        images_slug = cleaned_data.get('images_slug')
+        videos_slug = cleaned_data.get('videos_slug')
+        self.slug_fields = [vendors_slug, products_slug, preparations_slug, stories_slug, images_slug, videos_slug]  # noqa
+        if len(self.slug_fields) != len(set(self.slug_fields)):
+            raise forms.ValidationError('Error')
```

Comment:
This should be more helpful than error. Perhaps get duplicated items (perhaps with http://stackoverflow.com/a/15155286), and then say "can not use slug 'whatever' for multiple items!"